From ebe8c47943be0f51984af53dd4518c40d2c43edc Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 3 Feb 2022 00:53:38 +0800
Subject: [PATCH 33/36] arm64: allwinner: dts: axp803: add charging LED cell DT
 node

As we added the charging LED cell to the MFD driver, add a corresponding
DT node for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/axp803.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/axp803.dtsi b/arch/arm64/boot/dts/allwinner/axp803.dtsi
index 578ef368e2b4..2a37e1cea7ba 100644
--- a/arch/arm64/boot/dts/allwinner/axp803.dtsi
+++ b/arch/arm64/boot/dts/allwinner/axp803.dtsi
@@ -43,6 +43,12 @@ battery_power_supply: battery-power {
 		status = "disabled";
 	};
 
+	charger_led: charger-led {
+		compatible = "x-powers,axp803-charger-led",
+			     "x-powers,axp813-charger-led";
+		status = "disabled";
+	};
+
 	regulators {
 		/* Default work frequency for buck regulators */
 		x-powers,dcdc-freq = <3000>;
-- 
2.30.2

